File: ws/Perez/U2/WS21Inheritance/Zoo/src/main/java/ec/edu/espe/zoo/model/Condor.java
package ec.edu.espe.zoo.model;

import java.util.Date;

/**
 *
 * @author Carlos Perez,The Javas, DCCO-ESPE
 */
public class Condor extends Bird{

    protected String montain;

    public Condor(String montain, int id, String description, Date bornOnDate, char gender, Cage cage, int numberOfBones, boolean vetebrate) {
        super(id, description, bornOnDate, gender, cage, numberOfBones, vetebrate);
        this.montain = montain;
    }

    @Override
    public void feed(int food) {
        System.out.println("feeding a condor with dead animals");    
    }

    /**
     * @return the montain
     */
    public String getMontain() {
        return montain;
    }

    /**
     * @param montain the montain to set
     */
    public void setMontain(String montain) {
        this.montain = montain;
    }

    @Override
    public String toString() {
        return "Condor{" + "montain=" + montain + super.toString() + '}';
    }
    
    
}

File: hw/Vaca/u2/HW18ZooInPythonJavaScript/Zoo/ec.edu.espe.zoo.model/Condor.js
const Bird = require('./Bird');

class Condor extends Bird {
    constructor(id, description, dateOnborn, gender, numberOfBones, cage, wingSpan) {
        super(id, description, dateOnborn, gender, numberOfBones, cage, wingSpan);
    }
    
    toString() {
        return `Condor{wingSpan=${this.wingSpan}${super.toString()}}`;
    }
}

module.exports = Condor;
 